gnomad_qc/v4/create_release/create_release_sites_ht_exomes.py

```diff
@@ -147,13 +145,12 @@ def get_config(release_exists: bool = False):
         },
         "vep": {
             "ht": get_vep().ht(),
-            # TODO: drop 100% missing? Can add to the custom vep select function
             "path": get_vep().path,
             "select": ["vep"],
-            # Add in "custom_select" that drops insilicos
+            # Add in "custom_select" that drops in silicos and 100% missing VEP fields.
             "custom_select": custom_vep_select,
             # TODO: Update to have vep_csq_header -- should this be on the vep table
```

Comment:
Should we just add it in the VCF release code instead of here then?

Reply:
I think that's probably fine, but maybe check with @ch-kr to make sure she agrees

Reply:
this function drops fields from the VEP struct, correct? I'd be inclined to leave this here if that is the case, just to note that those are the fields we dropped in this release. however, if I read that function wrong, and this function is only related to VCF export, then I agree we don't need to keep this code here

Reply:
It drops the 100% missing fields, and the sift and polyphen annotations. The struct is kept and put onto the release HT.

Reply:
The vep_csq_header is only used in the VCF header. It's kinda unnecessary for the HT since you can do a ht.describe() to see hat fields are included

Reply:
oh no, I must have misread this -- I thought you were asking whether to keep the custom vep select here. agree that vep_csq_header doesn't need to be in the release HT code

Reply:
Sorry misread your comment. I guess I dont understand fully what people use the header for. I would think people would want it to reflect what is in the data?

Reply:
not at all, sorry, I misunderstood which line this comment was asking about. the vep csq header just describes which fields are in the vep field in the vcf export (and the order of the fields), so I agree that it doesn't need to get added to the release HT. for ukb, we added this field only when we were preparing to export data to vcf (so I agree it makes sense to move this to the VCF release code)